src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_VALUE = 50;
+    public static final int DEFAULT_RADIUS = 10;
+    public static final String DEFAULT_COLOR = "WHITE";
```

Comment:
Reference enum

Reply:
You mean use Color.WHITE instead of "WHITE" ? But I have to use String instead of Enum

Reply:
use Color.WHITE.name()